### ISSUE TITLE
Are you still maintaining this project??

### ISSUE THREAD

**Issue by kaltindas** [NaikSoftware/StompProtocolAndroid#51]

**Comment by forresthopkinsa** [comment]
I have a [fork](https://github.com/forresthopkinsa/StompProtocolAndroid/releases) of this project for compatibility with the new Non-Jack Android toolchain.

**Comment by NaikSoftware** [comment]
Yes but I have a little time for open sources projects..

**Comment by forresthopkinsa** [comment]
You deleted my comment? lol

**Comment by NaikSoftware** [comment]
@forresthopkinsa yes, because not need posting you url in all places, all peoples see all forks in forks section

**Comment by forresthopkinsa** [comment]
Okay, that's fair, I just assumed that the person asking this question hadn't seen the other forks, or else he wouldn't have asked. But you're right, I didn't need to post it here.
